### PR TITLE
Feat/mppt

### PR DESCRIPTION
**Pull request by ner24**
Will need to reupload to telemetry board
Don't forget this ^

### CODE REVIEW

File: Receiver/telemetryParser2.py [hunk 1]
```diff
@@ -128,7 +130,19 @@ def translateMsg(msgBytesAndTime: bytearray) -> tuple[str, str, dict, datetime,
             tzinfo=timezone.utc ) #msgData only contains last 2 digits of year so have to add 2000
         timeFetched = recievedMillisTime # update when data was last fetched
         print("GPS time is now: " + lastGPSTime.strftime("%Y-%m-%d %H:%M:%S"))
-    print(f'{msgSource}: {msgBody}')
+
+    #mppt
+    if canId == 1905:
```
Comment (owentmfoo) [inline Receiver/telemetryParser2.py]: There are two MPPT on the CAN, 0x771 and 0x772

File: Telemetry/src/CANApi
Comment (owentmfoo) [inline Telemetry/src/CANApi:1]: That also only got helper for 0x771? Are those helper functions used here?

**Comment by ner24** [comment]
Is 0x771 bytes the same format as 0x772?

Отправлено из Outlook для Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ner24** [comment]
Wait, is the config supposed to look like this?
image

**Comment by owentmfoo** [comment]
Yes, apologies for the miscommunication.

File: Receiver/__pycache__/telemetryParser2.cpython-310.pyc
Comment (owentmfoo) [inline Receiver/__pycache__/telemetryParser2.cpython-310.pyc:1]: Why are we adding pycache to git?

File: Receiver/__pycache__/telemetryStorer.cpython-310.pyc
Comment (owentmfoo) [inline Receiver/__pycache__/telemetryStorer.cpython-310.pyc:1]: Why are we adding pycache to git?

Comment (ner24) [reply Receiver/__pycache__/telemetryStorer.cpython-310.pyc:1]: because I wasn't paying attention. I've added a gitignore

**Review by owentmfoo — APPROVED** [review]

**Pull request merged**